src/main/java/com/example/enterprise/dao/ExpenseDAO.java

```diff
@@ -11,11 +11,20 @@
 
     List<Expense> allExpenses = new ArrayList<>();
 
+    /**
+     * Save an expense to the allExpenses arraylist
+     * @param expense an expense given by user
+     * @throws Exception
+     */
```

Comment:
Added some java doc for the methods. Java doc is necessary and a good coding convention to keep your code clean. 
To learn more about JavaDoc here is the link: https://www.tutorialspoint.com/java/java_documentation.htm

Reply:
It's good to have JavaDoc with a description in it.  :)